modules/apps/jaggery/emm/src/emm/units/user-listing/user-listing.hbs

```diff
@@ -5,94 +5,54 @@
         </div>
     </div>
     <!-- content -->
+    <div id="loading-content" class="col-centered">
+        <i class="fw fw-settings fw-spin fw-2x"></i>
+        &nbsp;&nbsp;&nbsp;
+        Loading users . . .
+        <br>
+    </div>
     <div class="container-fluid">
-        <table class="table table-striped table-hover list-table display responsive nowrap data-table table-selectable grid-view" id="user-grid">
-            <thead>
-            <tr class="sort-row">
-                <th class="no-sort"></th>
-                <th>By Name</th>
-                <th>By Username</th>
-                <th>By Email</th>
-                <th class="no-sort"></th>
-            </tr>
-            <tr class="filter-row filter-box">
-                <th class="no-sort"></th>
-                <th data-for="By Name" class="text-filter"></th>
-                <th data-for="By Username" class="text-filter"></th>
-                <th data-for="By Email" class="text-filter"></th>
-                <th class="no-sort"></th>
-            </tr>
-            <tr class="bulk-action-row">
-                <th colspan="5">
-                    <ul class="tiles">
-                        <li class="square">
-                            <a href="#" data-click-event="remove-form" class="btn square-element invite-user-link" data-toggle="modal" data-target="#modalDemo">
+        <span id="user-listing-status-msg"></span>
+        <div id="user-table">
+            <table class="table table-striped table-hover list-table display responsive nowrap data-table table-selectable grid-view"
+                   id="user-grid">
+                <thead>
+                <tr class="filter-row filter-box">
+                    <th colspan="4">
+                        <label class="wr-input-label" for="search-by-username">
+                            By Username
+                        </label>
+                        <input id="search-by-username" type="text" class="form-control"
+                               placeholder="Search By Username">
+                    </th>
+                    <th class="no-sort" style="vertical-align:bottom;">
+                        <span id="search-btn" class="wr-btn">
+                            &nbsp;&nbsp;&nbsp;&nbsp;Search&nbsp;&nbsp;&nbsp;&nbsp;</span>
```

Comment:
Noted.
And this can be found in all the buttons in EMM User Interface.This will be fixed in 2.0.1 patch and it currently in progress under the Jira[1].
[1]. https://wso2.org/jira/browse/EMM-1233